snowfakery/__init__.py

```diff
@@ -1,9 +1,7 @@
 from pathlib import Path
-from typing import Tuple, Union, Optional, Dict, TextIO, Sequence
-from contextlib import ExitStack
 
 from .plugins import SnowfakeryPlugin, lazy  # noqa
```

Comment:
I think you can remove the noqa comment now that it's included in `__all__`

Reply:
a4d8a7a